resources/js/Components/Alert.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <div class="alert" v-text="message"></div>
+</template>
+
+<script type="ts">
+export default {
+    data() {
+        return {
+            message: 'I am an alert.'
+        };
+    }
+};
+</script>
+
+<style>
```

Comment:
Perhaps we should try and see how sass works in the style section here, as I'm pretty sure we want to use sass also in our component styles.

Reply:
yes, sure. sass works out of the box if you add the `uselang` attribute to the tag like this `<style lang="scss">` will add it.